server/chat/backend/agent/tools/cloud_exec_tool.py

```
@@ -1231,7 +1231,8 @@ def _run_on_account(conn: dict) -> dict:
     })
 
 
-def cloud_exec(provider: str, command: str, user_id: Optional[str] = None, session_id: Optional[str] = None, provider_preference: Optional[str] = None, timeout: Optional[int] = None, output_file: Optional[str] = None, account_id: Optional[str] = None) -> str:
+def cloud_# FIX: 移除exec，改用安全方式
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)